care/emr/api/viewsets/tag_config.py

```
@@ -0,0 +1,47 @@
+    EMRUpdateMixin,
+)
+from care.emr.models.tag_config import TagConfig
+from care.emr.resources.tag.config_spec import (
+    TagConfigReadSpec,
+    TagConfigRetrieveSpec,
+    TagConfigUpdateSpec,
+    TagConfigWriteSpec,
+)
+
+
+class SpecimenDefinitionFilters(filters.FilterSet):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)